examples/language-modeling/run_clm.py

```diff
@@ -0,0 +1,323 @@
+# coding=utf-8
+# Copyright 2018 The Google AI Language Team Authors and The HuggingFace Inc. team.
+# Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Do we need the copyright to Google AI and NVIDIA? Are there some snippets taken from their codebases?

Reply:
No, it's a bad copy paste.